src/extension/content-script/batteries/Twitter.ts

```diff
@@ -7,7 +7,7 @@ declare global {
   }
 }
 
-const urlMatcher = /^https:\/\/(mobile.)?twitter\.com\/(\w+).*/;
+const urlMatcher = /^https:\/\/(mobile.)?x\.com\/(\w+).*/;
```

Comment:
this breaks things now on twitter.com 
it must support both

Reply:
how twitter.com is still accessible? resolved!

Reply:
I am always on twitter.com